### PR TITLE
implemented the loading screen to projects list

### PR DESCRIPTION
**Pull request by AlexGopal**

### CODE REVIEW

**Comment by mentor-tara** [comment]
You need to be a premium member for this task. Please head to [profy.dev](https://profy.dev#premium) and upgrade your account. Once you're done request another review.

**Review by mentor-tara — APPROVED** [review]
Go go go :smile:[](prolog-15)

**Comment by AlexGopal** [comment]
i wasn't sure where to put the loading screen but i see now this was more or less making a new component, i guess i initially treated it like fixing a bug because it uses the loading from the project-list. will use a new component in the future. 

i also need to work on matching the padding and spacing better to the designs.

**Comment by mentor-tara** [comment]
You need to be a premium member for this task. Please head to [profy.dev](https://profy.dev#premium) and upgrade your account. Once you're done request another review.

File: features/projects/components/project-list/project-list.module.scss [hunk 1]
```diff
@@ -15,3 +15,36 @@
...
+    margin-top: -300px;
+  }
+}
+
+.loader {
+  border: 8px solid #f9f5ff; /* Light grey */
+  border-top: 8px solid #7f56d9; /* Blue */
+  border-radius: 50%;
+  width: 48px;
+  height: 48px;
```
Comment (jkettmann) [inline features/projects/components/project-list/project-list.module.scss:36]: Here you could use the global SCSS variables to be consistent with the rest of the codebase

**Pull request merged**